iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -79,7 +78,7 @@ class ConversationListViewController: ViewController,
         = members.isEmpty ? .containMembers(userIds: [User.current()!.objectId!]) : .containOnlyMembers(members)
 
         let query = ChannelListQuery(filter: filter,
-                                     sort: [Sorting(key: .createdAt, isAscending: false)],
+                                     sort: [Sorting(key: .createdAt, isAscending: true)],
```

Comment:
Now that I'm more familiar with how to make custom layouts we don't need to do any more conversation order trickery.

The conversations come in oldest to newest, and are laid out as expected. Index 0 is on the far left.